live_flights_on_map.md

```diff
@@ -0,0 +1,125 @@
+## Documentation for live flight details on a map
```

Comment:
@arnabdutta73 give much more generic title something like `visualising live flight location on OSM` So that readers can relate to it much better.

Reply:
:+1: